docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -591,23 +591,85 @@ You can also nest several DTO :
             // Bind values to the object properties.
         }
     }
-	
+
     class AddressDTO
     {
         public function __construct(string $street, string $city, string $zip)
         {
             // Bind values to the object properties.
         }
     }
-	
+
 .. code-block:: php
 
     <?php
     $query = $em->createQuery('SELECT NEW CustomerDTO(c.name, e.email, NEW AddressDTO(a.street, a.city, a.zip)) FROM Customer c JOIN c.email e JOIN c.address a');
     $users = $query->getResult(); // array of CustomerDTO
-	
+
 Note that you can only pass scalar expressions or other Data Transfer Objects to the constructor.
 
+If you use your data-transfer objects for multiple queries but not necessarily with the same parameters,
+you can use named arguments or variadic arguments, add the ``named`` keyword in your DQL query before your DTO :
+
+.. code-block:: php
```

Comment:
Because of what precedes, I'm expecting to see a DQL query. I think you should reorganize this.
If you use your data transfer objects for multiple queries, and you would rather not have to
specify arguments that precede the ones you are really interested in, you can use named arguments.

Consider the following DTO, which uses optional arguments:

// code for CustomerDTO here

You can specify arbitrary arguments in an arbitrary order by using the named argument syntax, and the ORM will try to match argument names with the selected column names.
The syntax relies on the `NAMED` keyword, like so: